### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by zek**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "name": "laravel-notification-channels/netgsm",
+    "description": "NetGSM notification channel for Laravel 5.5 & 6.x"
+    "homepage": "https://github.com/laravel-notification-channels/netgsm",
```
Comment (atymic) [inline composer.json:4]: missing comma above this line

File: README.md [hunk 2]
```diff
@@ -1,2 +1,122 @@
-# netgsm
```
Comment (atymic) [inline README.md:1]: not sure if you want to have some kind of title?

**Review by atymic — COMMENTED** [review]
think some files may be missing?

Also, if you could remove the DSstore file :)

**Comment by atymic** [comment]
@zek not sure if you saw my comments here :)

**Comment by zek** [comment]
I'm sorry I have engagement :) I'm not available for a few days.

25 Oca 2020 Cmt, saat 03:41 tarihinde atymic <notifications@github.com>
şunu yazdı:

**Comment by atymic** [comment]
cc @zek when you have a chance :)

**Comment by atymic** [comment]
cc @zek

**Pull request closed without merging**